src/mito2/src/region_write_ctx.rs

```diff
@@ -243,4 +251,53 @@ impl RegionWriteCtx {
         self.version_control
             .set_sequence_and_entry_id(self.next_sequence - 1, self.next_entry_id - 1);
     }
+
+    pub(crate) fn push_bulk(&mut self, sender: OptionOutputTx, mut bulk: BulkPart) {
+        self.bulk_notifiers
+            .push(WriteNotify::new(sender, bulk.num_rows));
+        bulk.sequence = self.next_sequence;
+        self.next_sequence += bulk.num_rows as u64;
+        self.bulk_parts.push(bulk);
+    }
```

Comment:
So we won't write bulk parts to WAL now, right?

Reply:
> So we won't write bulk parts to WAL now, right?

Yes